include/boost/url/params.hpp

```diff
@@ -286,50 +234,45 @@ class params
     operator[](
         std::size_t pos) const;
 
-    /** Access first query parameter in the container
+    /** Access the first element
+
+        Access first query parameter in the container
 
         @return Reference to first query parameter in the container
 
      */
     const_reference
     front() const;
 
-    /** Access last query parameter in the container
+    /** Access the last element
```

Comment:
Yeah my fault for using the word Access, I just copied cppreference :)

Reply:
I changed it to "Return". But yes. cppreference uses the word "access". I actually like this word because it's a short way to also imply this is a reference.